module/cli_controller.py

```diff
@@ -37,12 +42,12 @@ def __init__(self, comm:BaseComm, command_file_list=None):
     """
     def load_tree(self):
         for file in self.command_file_list:
-            self.load_commands(file)
+            self.load_commands(file, self.root_node)
 
     """
     Loads a single JSON file into command structure
     """
-    def load_commands(self, file):
+    def load_commands(self, file, rootnode):
```

Comment:
rootnode does not comply with the style guide, should be either root_node or a different name